libs/designer/src/lib/core/templates/utils/helper.ts

```diff
@@ -52,24 +52,26 @@ export const getUniqueConnectors = (
   connections: Record<string, Template.Connection>,
   subscriptionId: string,
   location: string
-): Template.Connection[] => {
-  const allConnections = Object.values(connections);
-  return getUniqueConnectorsFromConnections(allConnections, subscriptionId, location);
+): Template.FeaturedConnector[] => {
+  const allConnectors = Object.values(connections).map((connector) => ({ ...connector, id: connector.connectorId }));
+  return getUniqueConnectorsFromConnections(allConnectors, subscriptionId, location);
 };
 
 export const getUniqueConnectorsFromConnections = (
```

Comment:
Can you double check if this is not getting called to list the connections in teh connections grid and also showing the connections summary in templateoverview and workflow summary tab?